DataFormats/FEDRawData/interface/FEDNumbering.h

```diff
@@ -108,6 +110,10 @@ class FEDNumbering {
      MAXSiPixeluTCAFEDID = 1349,
      MINTriggerUpgradeFEDID = 1350,
      MAXTriggerUpgradeFEDID = 1409,
+     MINCTPPSPixelsFEDID = 1462,
+     MAXCTPPSPixelsFEDID = 1466,
+     MINGEMFEDID = 1467,
+     MAXGEMFEDID = 1472,
```

Comment:
neither CTPPSPixel nor GEM changes in this file appear relevant to 80X.
Please remove.